### ISSUE TITLE
c3d Preprocessing

### ISSUE THREAD

**Issue by abskjha** [albertomontesg/keras-model-zoo#14]
Hi @albertomontesg,
What are the preprocessing steps? Like mean subtraction and normalization?

**Comment by SreenivasVRao** [comment]
@albertomontesg 

Can you answer? The original paper doesn't mention anything about this except:

> All video frames are resized into 128 × 171. This is roughly half resolution of the UCF101 frames. Videos are split into non-overlapped 16-frame clips which are then used as input to the networks. The input dimensions are 3 × 16 × 128 × 171.

Thanks for the Keras version! Great job!

**Comment by albertomontesg** [comment]
I know that the paper doesn't mention any preprocessing, but the typical preprocessing used in deep learning with images and video is substracting the mean of each of the color channels. Taking a look at the original repo it can be seen how the original model comes with the computed mean and the weights ([here](https://github.com/facebook/C3D/tree/master/examples/c3d_finetuning)) and also at the example to train from scratch is asked to compute the mean your dataset [here](https://github.com/facebook/C3D/blob/master/examples/c3d_train_ucf101/create_volume_mean.sh).
